src/SlashCommands.tsx

```diff
@@ -321,7 +321,7 @@ export const Commands = [
     }),
     new Command({
         command: 'jumptodate',
-        args: '<date>',
+        args: '<YYYY-MM-DD>',
         description: _td('Jump to the given date in the timeline (YYYY-MM-DD)'),
```

Comment:
Sounds fine, but we should probably remove the format from the description then, to not be redundant